utils/updateReadme.ts

```diff
@@ -0,0 +1,87 @@
+import * as core from '@actions/core';
+import * as toolCache from '@actions/tool-cache';
+import * as setupGcloud from '@google-github-actions/setup-cloud-sdk';
+import path from 'path';
+import * as Moustache from 'mustache';
+import * as fs from 'fs';
+
+interface GcloudResponseObject {
+  output: string; // stringified json object
+  commandString: string; // the gcloud command string to be run
+}
+
+const setupGcloudSDK = async (): Promise<GcloudResponseObject> => {
+  try {
+    const cmd = ['functions', 'event-types', 'list', '--format', 'json'];
+    // Setup latest Gcloud Version
+    const gcloudVersion = await setupGcloud.getLatestGcloudSDKVersion();
+
+    if (!setupGcloud.isInstalled(gcloudVersion)) {
+      await setupGcloud.installGcloudSDK(gcloudVersion);
+    } else {
+      const toolPath = toolCache.find('gcloud', gcloudVersion);
+      core.addPath(path.join(toolPath, 'bin'));
+    }
+    await setupGcloud.authenticateGcloudSDK();
+    const authenticated = await setupGcloud.isAuthenticated();
+    if (!authenticated) {
+      throw new Error('Error authenticating the Cloud SDK.');
+    }
+    const toolCommand = setupGcloud.getToolCommand();
+    const commandString = `${toolCommand} ${cmd.join(' ')}`;
+    // Print command string for logging
+    core.info(`Running: ${commandString}`);
+    // Run gcloud cmd.
+    const { output } = await setupGcloud.gcloudRun(cmd);
+    return {
+      output: output,
+      commandString: commandString,
```

Comment:
Do we need commandString? gcloudRun(cmd) will throw an error with `commandString` for non zero exit code.

Reply:
Ended up running gcloud command as a separate step and removed this code.